src/lib/Components/Bidding/Screens/RegistrationResult.tsx

```diff
@@ -38,9 +38,9 @@ const registrationCompleteMessage = {
 const registrationPendingMessage = {
   title: "Registration pending",
   description:
-    "You will receive an email from an Artsy specialist once your registration has been confirmed.\n" +
+    "Artsy is reviewing your registration and you will receive an email when it has been confirmed. Please email specialist@artsy.net with any questions.\n" +
```

Comment:
Do we usually wrap email addresses in a mailto link?

Reply:
Considering that we use a [`mailto` for the registration failed component](https://github.com/artsy/emission/blob/04af01b03b463cda21e620f67b42a040e43f7556/src/lib/Components/Bidding/Screens/RegistrationResult.tsx#L48), I think it's safe to assume that we should do so here. Can you make that change @yuki24?

Reply:
Okay, the description can actually be markdown and it is very easy to make it a link:

<img width="559" alt="Screen Shot 2019-08-02 at 10 51 50 AM" src="https://user-images.githubusercontent.com/386234/62378800-9e847e00-b513-11e9-965d-08063da6c6e9.png">

This link does not work on Emission due to an unhandled native event, but if I recall correctly this is handled properly in Eigen. We should QA this anwyay to make sure it works in the actual app.

Reply:
There seems to be a bug around the email click but there is another PR that addresses it: https://github.com/artsy/emission/pull/1769 so it's likely that it still crashes but it'll be fixed eventually.